### PR TITLE
Fabo/804 node version check

### LINKED ISSUE #804: Check node version against gaiacli version
UI Version: 0.6.2

Description:
The node we are connecting to might have another version of the SDK then we are using and therefor the LCD might produce unexpected results. We should check the version and display an error if the node is on another version.

### PR DESCRIPTION
**Pull request by faboweb**
Closes #804 

❤️ Thank you!

### CODE REVIEW

**Comment by faboweb** [comment]
Needs newer SDK

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * Release published on GitHub now including changes from CHANGELOG. @NodeGuy
 * Added ESLint errors: no-undef and no-unused-vars. @NodeGuy
 * Modal when a node doesn't signal any new blocks @faboweb
+* Checking known nodes round robin and signaling an error if no compatible and online nodes are available @faboweb
```
Comment (jolesbi) [inline CHANGELOG.md]: sorry @faboweb i don't understand this sentence 😢

Comment (faboweb) [reply CHANGELOG.md]: changed it

**Comment by faboweb** [comment]
I am OK with that, if we refactor out all the logic (like addressbook.js) and test those. (Issue for refactoring #856) And have basic E2E tests covering the functionality.
The main thread is too important to not test it in my oppinion.

**Comment by NodeGuy** [comment]
So shall we omit it now to get this PR through and refactor later?

**Pull request merged**